Gemfile

```diff
@@ -59,6 +59,11 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of chromedriver to run system tests with Chrome
   gem 'chromedriver-helper'
+  gem 'rails-controller-testing', '1.0.2'
```

Comment:
Coding convention:
1. Trong code ruby dùng nháy đôi em nhé
Nháy đơn sẽ dùng trong file js

2. Sau dấu phẩy thì 1 space thôi em

=> fix toàn bộ file này nhé